rotkehlchen/tests/api/test_ethereum_transactions.py

```diff
@@ -932,6 +944,22 @@ def query_transactions(rotki) -> None:
         gevent.joinall(rotki.greenlet_manager.greenlets)
         rotki.task_manager._maybe_decode_evm_transactions()
         gevent.joinall(rotki.greenlet_manager.greenlets)
+        with dbevents.db.user_write() as write_cursor:
+            # Add an event with random values. Important thing is that asset doesn't exist.
+            # We do it to check that this event will be skipped during deserialization
```

Comment:
Is it to skip during deserialization or to not fail when there is an even with unknown asset?

Reply:
Eh... both? This event won't be returned